### PR TITLE
Update inbound outbound messages and pagination

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@KaitCrawford please review:

Messages sent to user pagination
screenshot 2017-04-18 09 43 20

screenshot 2017-04-18 09 43 49


Outbound messages
screenshot 2017-04-18 09 44 08

File: ci/templates/ci/identities_detail.html [hunk 1]
```diff
@@ -241,24 +237,56 @@ <h2 class="dashhead-title">Identity Management</h2>
...
   </div>
 </div>
+<div class="pagination pagination-list">
+  {% if outbound_messages.previous %}
+    <a href="{{request.site.root_page.url}}?outbound_prev={{ outbound_prev }}" class="pagination__link pagination__link--previous">
+      <span class="pagination__link--arrow">&larr;</span> Prev
+    </a>
+  {% endif %}
+  {% if outbound_messages.next %}
+    <a href="{{request.site.root_page.url}}?outbound_next={{ outbound_next }}" class="pagination__link pagination__link--next">Next
```
Comment (KaitCrawford) [inline ci/templates/ci/identities_detail.html]: @Mitso I don't think the context has an `outbound_next` variable will `{{ outbound_next }}` cause problems if that variable isn't set?

**Comment by KaitCrawford** [comment]
@Mitso one question from me. if it's not an issue then :+1:

File: ci/templates/ci/identities_detail.html (hunk 1 above)
Comment (Mitso) [reply ci/templates/ci/identities_detail.html]: @KaitCrawford thanks.

That does not seem to work, variable {{outbound_prev}} return is empty.
 And this functions as expected {{request.site.root_page.url}}?outbound_next

**Review by KaitCrawford — APPROVED** [review]

**Pull request merged**